### DISCUSSION TITLE
Navigate to file from references list

### DISCUSSION THREAD

**Discussion by vuki656** [nvimdev/lspsaga.nvim#154] [Q&A] [answered]
Hey, love the plugin. Thanks for the awesome work. 

I have a question about `lsp_finder()`

When i use `gh`, i get this:

![image](https://user-images.githubusercontent.com/8104698/112664095-dadbbf00-8e59-11eb-85e9-8440906bf6d5.png)

Is it possible to open the selected file from that window? 

Right now when I press enter, it just goes to the next file in the list. I'm not sure if that is the intended behavior.

**Comment by glepnir** [comment, marked as answer]
check this kemap 

```
-- finder_action_keys = {
--   open = 'o', vsplit = 's',split = 'i',quit = 'q',scroll_down = '<C-f>', scroll_up = '<C-b>' -- quit can be a table
-- },
```
